tutorials/kafka-exactly-once/README.md

```diff
@@ -0,0 +1,49 @@
+# Kafka exactly-once example (Kafka 0.9 API)
+
+This application verifies exactly-once semantics by writing a defined sequence of input data to two Kafka
+output operators -- one that guarantees those semantics and one that does not,
+each writing to a different topic. It deliberately causes the intermediate pass-through
+operator to fail causing it to be restarted and some tuples to be reprocessed.
+Then a KafkaInputOperator reads tuples from both topics to verify that the former topic has no duplicates
+but the latter does and writes a single line to a HDFS file with the verification results
+of the following form:
+
+    Duplicates: exactly-once: 0, at-least-once: 5
+
+
+**DAG of this application:**
+
+    sequenceGenerator --> passthrough --> kafkaExactlyOnceOutputOperator
+                                                                  |
+                                                                  \--> kafkaOutputOperator (at-least-once)
+
+    kafkaInput --> validationToFile
+
+
+**Running the Application**
+
+***Run Test***
+
+The application can be run in local mode which will write the validation file
+to target/validation.txt
+
+
+***Run on Cluster***
+
+To run the application on a cluster a running Kafka service is needed.
+A local Kafka single-node instance can easily be deployed (see [kafka.apache.org/quickstart](https://kafka.apache.org/quickstart)).
+
+Kafka topics should be cleared after every run to get a right validation
+output line.

```

Comment:
Please provide instructions on how to clear the topics and which topics to clear.